components/tools/OmeroPy/src/omero/plugins/fs.py

```diff
@@ -272,6 +276,19 @@ def _configure(self, parser):
             "fileset",
             type=ProxyStringType("Fileset"))
 
+        logfile = parser.add(sub, self.logfile)
+        logfile.add_argument("fileset", type=ProxyStringType("Fileset"))
+        logfile.add_argument(
+            "filename",  nargs="?", default="-",
+            help="Local filename to be saved to. '-' for stdout")
```

Comment:
If pasted then consistency is good. Fine to push fixes to another PR on a rainy day.